### DISCUSSION TITLE
[✨] Ability to pass AbortSignal when submiting routeAction$ programatically

### DISCUSSION THREAD

**Discussion by amit13k** [QwikDev/qwik-evolution#150] [Proposals For Qwik]
### Is your feature request related to a problem?

Recently, AbortSignal support was added for server$. Since actions can also be triggered programmatically, it might be useful to implement the same for routeAction$ as well. 

### Describe the solution you'd like

Right now, the following code is accepted by the existing typescript definitions (but it doesn't work),
So, AbortSignal signal as an optional first parameter might be one of the ways to implement this feature. 
```
const signal = AbortSignal.timeout(3000);
const response = await greet.submit(signal, {
  name: "John",
});
```

Or maybe just follow the convention fetch/TRPC follow with { signal: ac.signal } as the last optional parameter.

Example from [fetch](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal)
```  
const res = await fetch(url, { signal: AbortSignal.timeout(5000) });
```

Example from [TRPC](https://trpc.io/docs/client/aborting-procedure-calls)
```
const query = proxy.userById.query('id_bilbo', { signal: ac.signal });
```

### Describe alternatives you've considered

just ignoring the output

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
